src/pytorch_lightning/loggers/logger.py

```diff
@@ -346,6 +346,14 @@ def __iter__(self):
         # if DummyLogger is substituting a logger collection, pretend it is empty
         yield from ()
 
+    def __getattr__(self, name: str) -> Callable:
+        """Allows the DummyLogger to be called with arbitrary methods, to avoid AttributeErrors."""
+
+        def method(*args, **kwargs):
```

Comment:
where is the info generated?

Reply:
Now only general info is generated when running in `fast_dev_run`, see [here](https://github.com/HallerPatrick/pytorch-lightning/blob/2833cc19e2f59f793be9cb91c42a79b6b8f51caf/src/pytorch_lightning/trainer/trainer.py#L594)